ansible/roles/fwu/tasks/main.yml

```
@@ -0,0 +1,56 @@
+  - name: FWU Deployment
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: deployment.yml.j2
+      state: "{{ 'present' if WITH_FWU is defined and WITH_FWU|bool else 'absent' }}"
+    tags:
+      - deployment
+
+  - name: FWU Service
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: svc.yml.j2
+      state: "{{ 'present' if WITH_FWU is defined and WITH_FWU|bool else 'absent' }}"
+    tags:
+      - service
+
+  - name: FWU ConfigMap
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: configmap.yml.j2
+      state: "{{ 'present' if WITH_FWU is defined and WITH_FWU|bool else 'absent' }}"
+    tags:
+      - service
```

wrong ansible Tag for the configmap
```suggestion
      - configmap
```